File: Project 14 - UIVisualEffectView/Project 14 - UIVisualEffectView/TableViewCell.m
//
//  TableViewCell.m
//  Project 14 - UIVisualEffectView
//
//  Created by Jamie on 2018/7/18.
//  Copyright © 2018年 Jamie. All rights reserved.
//

#import "TableViewCell.h"

@implementation TableViewCell

- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
{
    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
    if (self) {
        // Initialization code
        [self addSubview:self.picView];
    }
    return self;
}

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

-(UIImageView *)picView {
    if (!_picView) {
        _picView = [[UIImageView alloc] initWithFrame:CGRectMake(0, 10, [[UIScreen mainScreen] bounds].size.width, 80)];
    }
    return _picView;
}

@end

File: Project 08 - Pull To Refresh/Project 08 - Pull To Refresh/TableViewCell.swift
//
//  TableViewCell.swift
//  Project 08 - Pull To Refresh
//
//  Created by 一川 黄 on 08/11/2016.
//  Copyright © 2016 Jamie. All rights reserved.
//

import UIKit

class TableViewCell: UITableViewCell {

    @IBOutlet var titleLabel: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
